core/build.rs

```diff
@@ -19,13 +19,19 @@
 // DEALINGS IN THE SOFTWARE.
 
 fn main() {
-    prost_build::compile_protos(
-        &[
+    let cust: protobuf_codegen::Customize = Default::default();
+    protobuf_codegen::Codegen::new()
+        .pure()
+        .includes(&["src"])
+        .inputs(
+            &[
             "src/keys.proto",
             "src/envelope.proto",
             "src/peer_record.proto",
-        ],
-        &["src"],
-    )
-    .unwrap();
+            ]
+        )
+        .customize(cust.gen_mod_rs(true))
```

Comment:
It says [here](https://docs.rs/protobuf-codegen/latest/protobuf_codegen/struct.Customize.html#method.gen_mod_rs) that this is the default in version 3?

Reply:
It wasn't working when I was testing it and I don't [see](https://docs.rs/protobuf-codegen/latest/src/protobuf_codegen/customize/mod.rs.html#163-166) how where it defaults to true. I'll test it again.

Reply:
You were right about this so I'm removing it. I must have overlooked something.